apis/Google.Cloud.AspNetCore.DataProtection.SecretManager/docs/index.md

```diff
@@ -0,0 +1,17 @@
+## Getting Started
+
+The `Google.Cloud.AspNetCore.DataProtection.SecretManager` package allows storing ASP.NET keys in [Google Cloud Secret Manager](https://cloud.google.com/secret-manager).
+The keys can be shared across several instances of a webapp. Apps can share authentication cookies
+or CSRF protection across multiple servers
+
+Follow the examples below to see how the library can be integrated into your application.
+
+## Sample code
+```csharp
+public void ConfigureServices(IServiceCollection services)
```

Comment:
Thinking if we should have a snippet that can be referenced here. This will answer the obvious questions like what using directives to import etc. Please don't make any changes yet and wait for suggestions from Jon and Amanda.

Reply:
Yes, it would be better to have snippets so we know the code at least compiles - it doesn't need to be part of a test necessarily.

As it happens, I don't think this code *would* compile, because SecretManagerServiceClient is abstract...

Reply:
Added it.